packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -164,11 +203,21 @@ function handler(app: Application) {
               [MERGE_ON_GREEN_LABEL, MERGE_ON_GREEN_LABEL_SECURE],
               wp.state,
               wp.branchProtection,
-              context.github as any
+              wp.label,
+              wp.author,
+              github as any
             );
             if (remove || wp.state === 'stop') {
-              await handler.removePR(wp.url);
+              await handler.removeLabelAndReaction(
+                wp.owner,
+                wp.repo,
+                wp.number,
+                wp.label,
+                wp.reactionId,
+                github as any
+              );
             }
+            await handler.removePR(wp.url);
```

Comment:
Should this be in the `if(remove || wp.state === 'stop')` block?

Reply:
AWESOME catch! thanks!